libraries/Wire/src/Wire.cpp

```diff
@@ -411,37 +416,6 @@ void TwoWire::onRequest( void (*function)(void) )
   user_onRequest = function;
 }
 
-/**
-  * @brief  Change the size of the buffer.
-  * @param  buffer: pointer to the allocated buffer
-  * @param  length: number of bytes to allocate
-  * @retval pointer to the new buffer location
-  */
-uint8_t *TwoWire::allocateBuffer(uint8_t *buffer, size_t length)
-{
-  // By default we allocate BUFFER_LENGTH bytes. It is the min size of the buffer.
-  if(length < BUFFER_LENGTH) {
-    length = BUFFER_LENGTH;
-  }
-
-  buffer = (uint8_t *)realloc(buffer, length * sizeof(uint8_t));
-  return buffer;
-}
-
-/**
-  * @brief  Reset the buffer. Reduce its size to BUFFER_LENGTH.
-  * @param  buffer: pointer to the allocated buffer
-  * @retval pointer to the new buffer location
-  */
-uint8_t *TwoWire::resetBuffer(uint8_t *buffer)
-{
-  buffer = (uint8_t *)realloc(buffer, BUFFER_LENGTH * sizeof(uint8_t));
-  if(buffer != nullptr) {
-    memset(buffer, 0, BUFFER_LENGTH);
-  }
-  return buffer;
-}
-
```

Comment:
why moving to .h file ? maybe you can simply move it up in the C file ?

Reply:
Move it as `inline` function in the class to avoid function call

Reply:
inline is fine - but why not keep it in wire.cpp file ?